packager/signer/signer.go

```diff
@@ -89,6 +90,11 @@ var statusNotModifiedHeaders = map[string]bool{
 	"Vary":             true,
 }
 
+// A comma, as would appear in a Connection header. Comma-separation is defined
+// in https://tools.ietf.org/html/rfc7230#section-7, with OWS defined in
+// https://tools.ietf.org/html/rfc7230#appendix-B.
+var comma *regexp.Regexp = regexp.MustCompile(`[ \t]*,[ \t]*`)
```

Comment:
Is this identical to the regex defined in accept.go? Consider consolidating in one place.

Reply:
Yeah, it's identical. Filed #196.